.github/workflows/nodejs.yml

```diff
@@ -118,3 +118,41 @@ jobs:
         # $GITHUB_WORKSPACE is the path to your repository
         run: echo "::set-env name=NODE_PATH::$GITHUB_WORKSPACE/node_modules"
       - uses: wagoid/commitlint-github-action@v2
+
+  coverage:
+    name: Coverage - ${{ matrix.os }} - Node v${{ matrix.node-version }}
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [12.x]
+        webpack-version: [latest]
```

Comment:
We have many `if` on `webpack@4` and `webpack@5`, to cover them we need do tests for webpack@5 too

Reply:
Ideally run all tests with coverage, because it allows to catch problems on widnows/macos/linux, sometimes some checks are truthy on linux, but falsy on windows

Reply:
Okay, I will run the coverage at all platform + webpack v5 once this simple configuration works as expected.

Also, typescript test isn't working when ran using nyc and jest, but passes on jest only. Tested that locally as well.